portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,96 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Hide the comments form by default and fetch the login status from the servlet
+ * If the user is logged in, unhide the form
+ * If the user is not logged in, display a login link
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function checkLoginStatus(){
+    fetch('/home') // Sends a request to /home
+    .then(response => response.json()) // Parse the response as JSON
+    .then((object) => { // Reference the status in response
+        const status = object.status;
+        const url = object.url;
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+        if (status == 'true') {
+            document.getElementById('comment').style.display = 'block';
+            var text = 'logouthere';
+        }
+        else {
+            var text = 'loginhere';
+        }
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+        // Create the button
+        var button = document.createElement('button');
+        button.setAttribute('id','logbtn');
+        button.innerHTML = String(text);
+
+        // Append in login div
+        var login = document.getElementById('login');
+        login.appendChild(button);
+
+        // Add event handler
+        button.addEventListener('click', 
+            function() {
+                window.location.href = url;
+        });
+    });
+}
+
+/**
+ * Fetch the JSON string from the server and parse JSON
+ */
+function parseSomething(){
+
+    fetch('/comment')  // Sends a request to /data
+    .then(response => response.json()) // Parses the response as JSON
+    .then((object) => { // Reference the fields in Comment
+        object.sort(function(object1, object2) {
+            return object2.timestamp - object1.timestamp;
+        });
+        const commentsListElement = document.getElementById('comments-container');
+        commentsListElement.innerHTML = '';
+
+        var numberCommentsDisplay = document.getElementById('number-comment').value;
+        numberCommentsDisplay = Math.min(numberCommentsDisplay, object.length);
+
+        for (var i = 0; i < numberCommentsDisplay; i++) {
+            commentsListElement.appendChild(createListElement(sortedObjs[i]));
+        }
+    });
 }
+
+/** 
+ * Creates an <li> element containing text. 
+*/
+function createListElement(comment) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+
+  const contentElement = document.createElement('span');
+  contentElement.innerText = comment.content + ' by ' + comment.email;
+  
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerText = "goawayplease";
```

Comment:
I don't think it's a good name for the button. Prefer to make it clear and simple.
If you do want to multiple words, please add spaces between the words.

Reply:
All names of the buttons are changed! ;)